lms/djangoapps/bookmarks/models.py

```diff
@@ -0,0 +1,70 @@
+"""
+This File contains Model for Bookmarks.
+"""
+
+import json
+
+from django.contrib.auth.models import User
+from django.db import models
+
+from model_utils.models import TimeStampedModel
+
+from xmodule.modulestore.django import modulestore
+from xmodule_django.models import CourseKeyField, LocationKeyField
+
+
+class Bookmark(TimeStampedModel):
+    """
+    Bookmarks model.
+    """
+    user = models.ForeignKey(User)
+    course_key = CourseKeyField(max_length=255, db_index=True)
+    usage_key = LocationKeyField(max_length=255, db_index=True)
+    display_name = models.CharField(max_length=255, default="", help_text="Display name of XBlock")
+    _path = models.TextField(db_column='path', blank=True, help_text="JSON, breadcrumbs to the XBlock")
```

Comment:
We should actually be using the `JSONField` for this, which supports automatic serialization and deserialization.